aptos-move/aptos-vm/src/storage_adapter/executor_view_base.rs

```diff
@@ -43,6 +43,16 @@ impl<'s, S: StateView> TAggregatorView for ExecutorViewBase<'s, S> {
     ) -> anyhow::Result<Option<StateValue>> {
         self.0.get_state_value(state_key)
     }
+
+    fn get_aggregator_v2_value(
+        &self,
+        _id: &Self::IdentifierV2,
+        _mode: AggregatorReadMode,
+    ) -> anyhow::Result<AggregatorValue> {
+        Err(anyhow::Error::msg(
```

Comment:
Why error message here?
In data_cache.rs, we are calling `self.executor_view.get_aggregator_v2_value(id, mode)`. So, the executor view should have `get_aggregator_v2_value` implemented right?